tests/integration/test_random.py

```diff
@@ -21,33 +20,367 @@ def test_hash_deterministic_order(cluster):
     """
 
     cluster.create(3)
-    #cluster.config_set('lua-replicate-commands', 'no')
 
     script = cluster.node(1).client.register_script("""
 -- Populate hash KEYS[1] with ARGV[1] fields, then assign each field
--- a value corresponding to its index in HKEYS
-local key = KEYS[1]
+-- a value corresponding to its index in result of cmd
+local key1 = "test"
```

Comment:
Well behaved Lua scripts need to only access keys listed in the KEYS array ("announced" keys).

Reply:
ok.  easy enough to fix.  will do that.